File: 1. 단계별로_풀어보기/7. 2차원 배열/2_2566.js
// 입력
const fs = require('fs')
const inputArr = fs.readFileSync(0, 'utf-8').toString().trim().split('\n')

const M = inputArr.map(row => row.split(' ').map(Number))

// 최대값 찾기
let max_row = 0
let max_col = 0
let maxValue = -1

M.forEach((row, rowIdx) => {
  row.forEach((val, colIdx) => {
    if (maxValue < val) {
      maxValue = val
      max_row = rowIdx
      max_col = colIdx
    }
  })
})

// 결과값 출력
process.stdout.write(`${maxValue}\n`)
process.stdout.write(`${max_row + 1} ${max_col + 1}`)

File: 1. 단계별로_풀어보기/1~9/7. 2차원 배열/2_2566.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

M = list(list(map(int, input().split())) for i in range(9))

max_row = 0
max_col = 0
maxValue = -1

for idx_row in range(9):
  for idx_col in range(9):
    if maxValue < M[idx_row][idx_col]:
      maxValue = M[idx_row][idx_col]
      max_row = idx_row
      max_col = idx_col

output(f"{maxValue}\n{max_row + 1} {max_col + 1}")